apps/codeforafrica/package.json

```diff
@@ -59,6 +59,7 @@
     "js-yaml": "^4.1.0",
     "jsdom": "^21.1.2",
     "marked": "^4.3.0",
+    "mongodb": "^5.7.0",
```

Comment:
Since we're already using Payload and it uses [`mongoose`](https://mongoosejs.com), can't we use the same client **_if_** we're going to go down this way?